TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/RouteToProfessionalStatus.cs

```diff
@@ -92,15 +92,27 @@ public static RouteToProfessionalStatus Create(
             TrainingProviderId = trainingProviderId,
             TrainingSubjectIds = trainingSubjectIds ?? []
         };
-        route.RefreshExemptFromInductionDueToQtsDate(routeType.ProfessionalStatusType);
+
+        if (routeType.ProfessionalStatusType == ProfessionalStatusType.QualifiedTeacherStatus)
+        {
+            route.RefreshExemptFromInductionDueToQtsDate();
```

Comment:
this seems to be doing the same thing as line 109 unless I'm not seeing something?